app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -8,6 +8,8 @@ class GetUspsProofingResultsJob < ApplicationJob
   IPP_EXPIRED_ERROR_MESSAGE = /More than (?<days>\d+) days have passed since opt-in to IPP/
   IPP_INVALID_ENROLLMENT_CODE_MESSAGE = 'Enrollment code %s does not exist'
   IPP_INVALID_APPLICANT_MESSAGE = 'Applicant %s does not exist'
+  IPP_BAD_SPONSOR_ID_MESSAGE = /sponsorID \d is not registered as an IPP client/
+  IPP_SPONSOR_ID_NOT_FOUND_MESSAGE = /Sponsor for sponsorID \d not found/
```

Comment:
Are we guaranteed that the number here is only one character long? (i.e. a number between 0 and 9)

Reply:
No, it could be longer.   I updated this.  Thanks for catching that.

Reply:
Yes, good catch!